test/test_mbsubmit.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# This file is part of beets.
+# Copyright 2015, Adrian Sampson and Diego Moreda.
+#
+# Permission is hereby granted, free of charge, to any person obtaining
+# a copy of this software and associated documentation files (the
+# "Software"), to deal in the Software without restriction, including
+# without limitation the rights to use, copy, modify, merge, publish,
+# distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so, subject to
+# the following conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+
+from __future__ import (division, absolute_import, print_function,
+                        unicode_literals)
+
+from test._common import unittest
+from test.helper import capture_stdout, control_stdin, TestHelper
+from test.test_importer import ImportHelper, AutotagStub
+from test.test_ui_importer import TerminalImportSessionSetup
+
+
+class MBSubmitPluginTest(TerminalImportSessionSetup, unittest.TestCase,
+                         ImportHelper, TestHelper):
+    def setUp(self):
+        self.setup_beets()
+        self.load_plugins('mbsubmit')
+        self._create_import_dir(2)
+        self._setup_import_session()
+        self.matcher = AutotagStub().install()
+
+    def tearDown(self):
+        self.unload_plugins()
+        self.teardown_beets()
+
+    def test_xxxxprint_tracks_output(self):
```

Comment:
Yes, there is a powerful reason: my sometimes absent mind! :hammer:

On a more serious note, I did notice just after submitting the pull request and already fixed it on my local copy. The current code was still meant to have a final cleanup phase on my end, that will also address the issue with the "non-full sentence" comments - sorry about that.